src/utils/routes.ts

```diff
@@ -52,11 +49,22 @@ export const getRoutesFromPath = (path: unknown, functionName: string, methods:
     // for both `/foo` and `/foo/`.
     const normalizedRegex = `^${regex}\\/?$`
 
-    return [{ pattern: path, expression: normalizedRegex, methods }]
+    return { pattern: path, expression: normalizedRegex, methods }
   } catch {
     throw new FunctionBundlingUserError(`'${path}' is not a valid path according to the URLPattern specification`, {
       functionName,
       runtime: RUNTIME.JAVASCRIPT,
     })
   }
 }
+
+export const getRoutes = (input: unknown, functionName: string, methods: string[]): Route[] => {
```

Comment:
NIT: would love a small jsdoc that explains the parameters and the method. So that a tipsy on call engineer at 2am in the morning can understand it as well :D

Reply:
Done.